docs/configuration-reference/components/linkerd.md

```diff
@@ -0,0 +1,58 @@
+# Linkerd configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Attribute reference](#attribute-reference)
+* [Applying](#applying)
+* [Deleting](#deleting)
+
+## Introduction
+
+Linkerd is an open-source, cloud-native and lightweight service mesh. Linkerd uses sidecar proxy (linkerd-proxy) to create a mesh among all pods. Control plane components are installed in linkerd namespace.
+
+> **NOTE**: This is an **unsupported and experimental component**. It is not recommended to use this in production. The UX of this component can change anytime as it is tested overtime. If there are fundamental problems with the component then it could be removed without prior notice.
+
+## Prerequisites
+
+* A Lokomotive cluster accessible via `kubectl`.
```

Comment:
Does this component work only on Lokomotive clusters?

Reply:
It can work on other clusters as well. But there is no testing on them so can't claim it works everywhere.

Reply:
Fair enough. Perhaps we could create an issue to change it for all components.